resources/views/layouts/app.blade.php

```diff
@@ -1,62 +0,0 @@
-@extends('layouts.base')
-
-@section('body')
-<div id="app">
-    <nav id="nav-main" class="navbar navbar-expand-lg fixed-top navbar-light bg-light">
```

Comment:
Everything in this `<nav>` element is moved without any changes into `base.blade.php`.